app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/AddCoInsuredBottomSheetContent.kt

```diff
@@ -198,6 +202,9 @@ private fun FetchFromSsnFields(
         ssnInput = it
       },
       errorText = errorMessage,
+      visualTransformation = visualTransformation@{ annotatedString: AnnotatedString ->
```

Comment:
Can just remove the `visualTranformation@` part since we do not use it here to do a `return@visualTransformation` here somewhere